### PR TITLE
Crypto Track Tutorial

### PR DESCRIPTION
**Pull request by eyuelberga**

### CODE REVIEW

File: crypto_track/blog.md [hunk 1]
```diff
@@ -0,0 +1,574 @@
+# Crypto Track: Cryptocurrency tracking web application with instant email notifications
+
+## Background
+
+[Crypto Track](https://github.com/eyuelberga/CryptoTrack/) is a web application to help track market changes on your favorite cryptocurrency and receive instant email notifications. Cryptocurrency use is rapidly expanding, and many people are putting their money and resources into it. However, its volatility sets it apart from other assets. Even though staying current with market trends might be time-consuming, there are also considerable benefits to doing so. Crypto Track essentially aims to make this a lot easier by notifying subscribers when the desired change is reflected on the market.
```
Comment (shreythecray) [inline crypto_track/blog.md:11]: Beautiful introduction!

File: crypto_track/blog.md [hunk 2]
```diff
@@ -0,0 +1,574 @@
+# Crypto Track: Cryptocurrency tracking web application with instant email notifications
+
+## Background
```
Comment (shreythecray) [inline crypto_track/blog.md:9]: I'd recommend adding either screenshots/gifs or embedding your demo video at the top.

File: crypto_track/blog.md [hunk 3]
```diff
@@ -0,0 +1,574 @@
...
+
+There are a lot of utility functions we need to create for working with the Supabase database and also fetching latest market data. You can see all of them on [GitHub](https://github.com/eyuelberga/CryptoTrack/tree/main/cron/src). The most important one we need to focus on is the `notification.js` file. This holds the logic for how we notify new market changes to subscribers and send emails using the Courier API. 
+
+
+The main function here is `notify`:
+
+```js
+
+const notify = async (supabase) => {
+    // get all notifications
```
Comment (shreythecray) [inline crypto_track/blog.md:367]: Love how you commented your code.

File: crypto_track/blog.md [hunk 4]
```diff
@@ -0,0 +1,574 @@
...
+      )
+    $$
+  );
+ 
+
+```
+
+## Conclusions
+
+And we are done, I hope you enjoyed this tutorial. You can find the full source code on [Github](https://github.com/eyuelberga/CryptoTrack/) and also check about the demo [here](https://eyuelberga.github.io/CryptoTrack/).
```
Comment (shreythecray) [inline crypto_track/blog.md]: What are some challenges you faced while building this? How would you suggest the readers challenge themselves to take this project to the next level? What was your experience like using Courier?

File: crypto_track/blog.md [hunk 5]
```diff
@@ -0,0 +1,574 @@
...
+
+## About the Author
```
Comment (shreythecray) [inline crypto_track/blog.md:568]: I have very little feedback overall, this was a really well written tutorial. I'm excited to share with with our community.

**Review by shreythecray — CHANGES_REQUESTED** [review]

File: crypto_track/blog.md (hunk 1 above)
Comment (eyuelberga) [reply crypto_track/blog.md:11]: Thank you!

File: crypto_track/blog.md (hunk 2 above)
Comment (eyuelberga) [reply crypto_track/blog.md:9]: Done. I have added a link to the demo video

File: crypto_track/blog.md (hunk 3 above)
Comment (eyuelberga) [reply crypto_track/blog.md:367]: Thanks. Without them, I would be lost reading my own code the next day.

File: crypto_track/blog.md (hunk 4 above)
Comment (eyuelberga) [reply crypto_track/blog.md]: I've added those points conclusion

File: crypto_track/blog.md (hunk 5 above)
Comment (eyuelberga) [reply crypto_track/blog.md:568]: Thank you. I'm also eager to see how it turns out.

**Pull request merged**